Makefile

```diff
@@ -1,6 +1,7 @@
 DESTDIR :=
-BINDIR := /usr/local/bin
-SYSCONFDIR := /etc
+PREFIX := /usr/local
+BINDIR := $(PREFIX)/bin
```

Comment:
Right I was somewhat hoping this would be for another time, but might as might do it properly now. We also need to make sure the runtime is opening the right config file by default, the one in `$(SYSCONFDIR)/clear-containers`.